homeassistant/components/drop_connect/coordinator.py

```diff
@@ -0,0 +1,66 @@
+"""DROP device data update coordinator object."""
+from __future__ import annotations
+
+from collections.abc import Callable
+import logging
+from typing import TYPE_CHECKING
+
+import attr
+from dropmqttapi.mqttapi import DropAPI
+
+from homeassistant.components import mqtt
+from homeassistant.components.mqtt import ReceiveMessage
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import CONF_DATA_TOPIC, CONF_DEVICE_TYPE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class DROPDeviceDataUpdateCoordinator(DataUpdateCoordinator):
+    """DROP device object."""
+
+    _drop_api: DropAPI | None = None
+    _unsubscribe_callback: Callable[[], None] | None = attr.ib()
```

Comment:
```suggestion
    _unsubscribe_callback: Callable[[], None] | None = None
```

Reply:
This change became moot after moving the MQTT subscription out of the coordinator in 414b4dbe5599a4135dc0f54b831d09493c0368d4.